_posts/en/newsletters/2018-07-24-newsletter.md

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Bitcoin Optech Newsletter #5'
+permalink: /en/newsletters/2018/07/24/
+name: 2018-07-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+version: 1
+---
+This week's newsletter includes news about a new method under
+development for describing which outputs you'd like to find, the merge
+of partially-signed Bitcoin transactions into Bitcoin Core, and several
+other notable Bitcoin Core merges.
+
+## Action items
+
+- Bitcoin Core [0.16.2RC2][] released for testing in preparation for a
+  maintenance release that will provide bugfixes and backports.
+  Community testing is highly appreciated.  Note, there was no RC1 due
+  to a metadata problem being detected and corrected after RC1 was
+  tagged but before it was released.
+
+## Dashboard items
+
+- Transaction fees are lower than they were this time last week.  Anyone
+  who can wait 12 or more blocks for confirmation can reasonably pay the
+  default minimum feerate.  It's a good time to [consolidate UTXOs][].
+
+- The number of [native segwit outputs][p2shinfo bech32] had been
+  increasing steadily over time, but dropped by about 400,000 (80%) this
+  week, probably as part of a consolidation by an exchange.  The average
+  number of new native segwit outputs created per hour remains
+  relatively constant, indicating no obvious decrease in adoption.
+  (Remember that if your service accepts payments to native segwit
+  addresses, you'll [save up to 16%][native segwit save] in fees when
+  you spend them later compared to using P2SH-wrapped segwit, which can
+  itself [save 26% or more][p2sh segwit save] compared to legacy
+  outputs.)
+
+## News
+
+- **Bitcoin Optech publicly announced:** we received great coverage in
+  [Bitcoin Magazine][announce bmag], [Coindesk][announce cdesk], and several
+  other publications.  This wouldn't have been possible without the
+  support of our founding sponsors and member companies.  Thank you!
+
+- **Coin selection RPC unlikely:** In Bitcoin Core's [weekly
+  meeting][bcc meeting 7/19], Andrew Chow raised the possibility of
+  creating an RPC that would allow users to pass in information about a
+  transaction they wanted to create, including a list of available
+  inputs, and receive back a list of which inputs would be selected by
+  the Bitcoin Core wallet's coin selection algorithm.
+
+    Meeting participants were mostly opposed to providing this feature,
```

Comment:
added "...as an RPC"

Reply:
I reverted this.  My read on the discussion is that most discussion participants were against it being in Bitcoin Core whatsoever, and that they additionally thought that (if it was done), it should be done as a library---probably a third-party library.  Just to check, I rescanned that discussion; here were the participants:

achow, wumpus, gmaxwell, jonasschnelli, sipa, jnewbery

Here are one comment per person against making it part of Bitcoin Core.  Note, I think the comment from sipa might be more a response to wumpus than an opinion of his own.

    19:20:06 <gmaxwell> But the argument I was making above is also an argument against a library: pressure to maintain a stable interface to this would be harmful to the project.
    19:22:00 <wumpus> it's fine, they can make a library out of it themselves
    19:29:53 <jonasschnelli> Agree with wumpus: I guess its a one-day job to extract the coin selection code and create a library out of it...
    19:31:58 <sipa> wumpus: perhaps once the code is sufficiently encapsulated, someone else can librarify that and maintain it

So that's either 3/6 or 4/6 participants suggesting whatever it is should be outside of Bitcoin Core, and with 2/6 being strongly opposed to including it in Bitcoin Core.  I think that's sufficient opposition that (in the absence of other evidence) we shouldn't hold out hope that this could be part of a Bitcoin Core-provided library.

Reply:
Yes, that's fair.